components/backends/trtllm/performance_sweeps/submit_disagg.sh

```diff
@@ -293,6 +293,46 @@ main() {
                 run_32_gpus_mtp
             fi
             ;;
+        "pareto")
+            # 1k/1k
+            export ISL=1024
+            export OSL=1024
+
+            # 1k/1k mtp=on
```

Comment:
Either we should conditionalize this on the basis of `mtp=on` or `mtp=off` provided in $1.
Or we should relax the check in L276.

Reply:
conditionalized it